desktop-widgets/mainwindow.ui

```diff
@@ -53,7 +62,7 @@
      <x>0</x>
      <y>0</y>
      <width>861</width>
-     <height>23</height>
+     <height>30</height>
```

Comment:
and stuff like this always worries me. You remove things, yet the height value went up. Hmm?